### DISCUSSION TITLE
Architecture Question for data redundancy

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#13618] [Unsupported Versions]
I've been using rather beefy server in standalone mode for some time.  It's fine for the small network that we have and I'm content with the job it's doing.  But, the one thing I don't have is a solution for reduncancy (I know, some people don't think that it's important, but my CEO and CFO do).  Could I simply fire up another copy of SO and configure it as a search node and then add it to my grid?  

Is this a reasonable solution?  Would I need something more complex?

**Comment by reyesj2** [comment]
You can add a searchnode, but for it to replicate the data you will need to configure elasticsearch replicas.

That can be set for all new indices created via ```SOC > Administration > Configuration > elasticsearch > index_settings > global_overrides > index_template > template > settings > index > number_of_replicas```

If you need existing data to also be replicated you will need to update the index settings directly example via cli
```so-elasticsearch-query .ds-logs-zeek-so-2024.09.09-000001/_settings -d '{"index":{"number_of_replicas":1}}' -XPUT```

Either setting would be updated _after_ you've joined your new searchnode
